html/employment.html

```diff
@@ -173,10 +173,7 @@
 
         const bridgeToken = await apiRequests.getBridgeToken()
         const bridge = CitadelBridge.init({
-          clientName: 'Citadel Quickstart',
           bridgeToken: bridgeToken.bridge_token,
-          product: '{{ product_type }}',
-          trackingInfo: 'any data for tracking current user',
```

Comment:
Why do. you remove trackingInfo here but don't add it when you create bridge token?

Reply:
tracking info was there when I first started working with the quickstart, but I don't really think it's necessary to have in the quickstart, which is why I didn't add it into the bridge token call

Reply:
Would be good to have it in the quickstart